Objects Automated Test - LPS-152054 | master

timpak opened the pull request:
https://issues.liferay.com/browse/LPS-152054

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

timpak commented:
Hey @evanilson-santana, I made some minor changes to your test. 

1. I combined your commits to remove the separate SF commit. In the future, please include the SF changes in the commit that caused the SF error.
2. I removed the step to mark the view as default. From my testing, it doesn't seem necessary for this test.
3. I added an additional assertion to assert the Save button is disabled. That way we are actually checking the default sort cannot be saved without a column.

Let me know if you have any questions. Overall, good job!

timpak commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

The pull request will automatically be forwarded to the user `brianchandotcom` if the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-38.liferay.com/job/test-portal-source-format/338/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-source-format/builds/338/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/5445d064e5ec726bf000ff26e8bb357ed99b2dc5">5445d064e5ec726bf000ff26e8bb357ed99b2dc5</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/timpak/liferay-portal/tree/LPS-152054">LPS-152054</a><br/>Branch GIT ID: <a href="https://github.com/timpak/liferay-portal/commit/489fd0e89ed0ab1d0e10a99d76e3a1e2fd972125">489fd0e89ed0ab1d0e10a99d76e3a1e2fd972125</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-38.liferay.com/job/test-portal-source-format/338/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-source-format/builds/338/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-38.liferay.com/job/test-portal-source-format/338//consoleFull)

liferay-continuous-integration commented:
Pull request has been successfully forwarded to https://github.com/brianchandotcom/liferay-portal/pull/117612
[Console](https://test-1-38.liferay.com/job/test-portal-source-format/338//consoleFull)

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-38.liferay.com/job/test-portal-source-format/338/">test-portal-source-format#338</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-source-format/builds/338/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/timpak/liferay-portal/pull/214">timpak#214</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1852616535">[master] ci:test:sf - timpak &gt; timpak - PR#214 - 2022-05-18[00:30:07]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-38.liferay.com/job/publish-testray-report/1200/">publish-testray-report#1200</a><br/></div></div>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest(master)/503/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/503/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
This pull request already has open forwarded pull request(s):
https://github.com/brianchandotcom/liferay-portal/pull/117612

Pull request will not be forwarded to `brianchandotcom`.
[Console](https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest(master)/503//consoleFull)

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest(master)/503/">test-portal-acceptance-pullrequest(master)#503</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/503/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/timpak/liferay-portal/pull/214">timpak#214</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1852619969">[master] ci:test:relevant - timpak &gt; timpak - PR#214 - 2022-05-18[01:48:18]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-38.liferay.com/job/publish-testray-report/1207/">publish-testray-report#1207</a><br/></div></div>
